ray-operator/controllers/ray/common/pod.go

```diff
@@ -146,8 +166,11 @@ func BuildPod(podTemplateSpec v1.PodTemplateSpec, rayNodeType rayiov1alpha1.RayN
 			args = cont
 		}
 
+		// TODO (@DmitriGekhtman) Always append `block`?
```

Comment:
maybe not the greatest idea if the start command is specified as `./setup_env_and_ray_start.sh` or something.

Reply:
Context: The entrypoint can be configured either using rayStartParams or by explicitly setting a custom entrypoint (like `./setup_env_and_ray_start.sh`).

Modifying `setMissingRayStartParams` wouldn't override a custom entry-point.